1.9/administration/installing/custom/configuration-parameters.md

```diff
@@ -183,12 +183,17 @@ This required parameter specifies a YAML nested list (`-`) of DNS resolvers for
 
 This parameters specifies whether to enable the DC/OS proxy.
 
-*  `use_proxy: 'false'` Do not configure DC/OS [components](/docs/1.9/overview/components/) to use a custom proxy. This is the default value.
-*  `use_proxy: 'true'` Configure DC/OS [components](/docs/1.9/overview/components/) to use a custom proxy. If you specify `use_proxy: 'true'`, you can also specify these parameters:
-    **Important:** The specified proxies must be resolvable from the provided list of [resolvers](#resolvers).
+*  `use_proxy: 'false'` Do not configure DC/OS [components](/docs/1.8/overview/components/) to use a custom proxy. This is the default value.
+*  `use_proxy: 'true'` Configure DC/OS [components](/docs/1.8/overview/components/) to use a custom proxy. If you specify `use_proxy: 'true'`, you can also specify these parameters:
```

Comment:
On this line and the one above, why did you change the links to 1.8 docs, even though this page is for 1.9?